webpack.config.js

```diff
@@ -14,13 +14,13 @@ module.exports = (env = {}) => {
   );
   console.log(
     `Training material folder: '${trainingMaterialFolder}'.`,
-    `This can be changed using '--env.material=<relative path to training material folder>'.`
+    `This can be changed using '--env material=<relative path to training material folder>'.`
```

Comment:
The option is just `--material`, no? (And `--slug` for the slug)

```shell-session
$ sensei --help
sensei.js <command>

Commands:
  sensei.js serve [material] [slug]  serve locally, with live-reloading
  sensei.js pdf [material] [slug]    build PDFs
  sensei.js build [material] [slug]  build the static web site

Options:
  --version      Show version number                                   [boolean]
  --material     Path to the folder containing the training material. Defaults
                 to the current working directory.       [string] [default: "."]
  --slug         Training name used in PDF files and HTML page titles. Defaults
                 to the name of the current working directory.
                                 [string] [default: "formation-kubernetes-user"]
  --slideWidth   Forwarded to Reveal.js. See
                 https://revealjs.com/presentation-size/.
                                                        [number] [default: 1420]
  --slideHeight  Forwarded to Reveal.js. See
                 https://revealjs.com/presentation-size/.[number] [default: 800]
  --help         Show help                                             [boolean]
```

Reply:
These warnings are emitted from `webpack.config.js`, so as far as it is concerned, the caller is probably using the Webpack CLI, which uses this syntax. And indeed if you look at the `npm start` script, it uses the webpack CLI, not the sensei CLI. This is a historic artifact, from the time when sensei was not a CLI yet. I've opened issue #81 about cleaning up this situation and use the sensei CLI everywhere, and I hope to tackle it soon.